code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -139,8 +146,14 @@
 	result += "<a href='?src=[REF(owner)];common=give_objective'>Force add objective</a><br>"
 	return result
 
+/// proc that generates the traitors replacement uplink code and radio frequency
+/datum/antagonist/traitor/proc/generate_replacement_codes()
+	replacement_uplink_code = "[pick(GLOB.phonetic_alphabet)] [rand(10,99)]"
+	replacement_uplink_frequency = sanitize_frequency(rand(MIN_FREQ, MAX_FREQ), free = FALSE, syndie = FALSE)
```

Comment:
Unless I am misunderstanding something `MIN_FREQ > MAX_FREQ` includes problematic values `1447` and `1459`, AI Private and Common Radio specifically.
It'd be really funny if a traitor had to say their code over the common channel but doesn't make a lot of sense. You might want to set a different minimum define which excludes these?

Reply:
made a new define that excludes common and ai so now its just frequencies between 146.1 and 148.9